src/Nox.Types/Types/Distance/Distance.cs

```diff
@@ -1,74 +1,175 @@
 ﻿using Nox.Types.Common;
+using System;
 using System.Collections.Generic;
-
+using System.Globalization;
 namespace Nox.Types;
 
-public class Distance : Measurement<Distance, DistanceUnit>
+/// <summary>
+/// Represents a Nox <see cref="Distance"/> type and value object.
+/// </summary>
+public class Distance : ValueObject<QuantityValue, Distance>
```

Comment:
Why did this change since `Measurement` already implements a lot of the methods you have here?

Reply:
There were a few reasons and one of them was that it was difficult to add type options and enum as unit for the type options with Measurement as base class. Therefore we decided to remove the inheritance.